src/tls13.c

```diff
@@ -1042,6 +1042,71 @@ int DeriveMasterSecret(WOLFSSL* ssl)
 #define RESUMPTION_LABEL_SZ         10
 /* Resumption label for generating PSK associated with the ticket. */
 static const byte resumptionLabel[RESUMPTION_LABEL_SZ+1] = "resumption";
+
+static int HKDFPSK(byte* okm, word32 okmLen, const byte* prk, word32 prkLen,
+    const byte* protocol, word32 protocolLen, const byte* info, word32 infoLen,
+    int digest, void* heap)
+{
+    int ret = 0;
+    int idx = 0;
+    byte* data;
+
+    (void)heap;
+    if (infoLen > 255)
```

Comment:
Let's add a "MAX" macro. `#define TLS13_TICKET_NONCE_MAX_SZ 255`

Reply:
:+1: .
I also removed MAX from `TLS13_TICKET_NONCE_STATIC_MAX_SZ` to avoid confusion